modules/gitops-deploy-resources-different-namespaces.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="deploy-resources-different-namespaces_{context}"]
+= Deploy resources to a different namespace
+
+[role="_abstract"]
+To grant Argo CD the permissions to manage resources in multiple namespaces, configure the namespace with a label "Argo CD.argoproj.io/managed-by".
+
+.Procedure
+. Create the following yaml.
++
+[source,yaml]
+----
+  apiVersion: v1
+kind: Namespace
+metadata:
+  name: namespace_name // new namespace to be managed by an existing Argo CD instance
+  labels:
+    argocd.argoproj.io/managed-by: instance_name // namespace of the Argo CD instance
+----
+. Run the following command to create/configure the namespace.
++
+[source,terminal]
+----
+$ oc apply -f namespace.yaml
+----
```

Comment:
IMHO, this could be a single `oc` command, e.g.

```shell
oc label namespace <name of namespace> argocd.argoproj.io/managed-by=<instance name>
```

Reply:
Also, we should point out that `<instance name>` is actually the name of the namespace where the instance is installed to, and not the name of the `ArgoCD` resource in that namespace.

For example, if you have an `ArgoCD` instance named `argocd` in the namespace `gitops-manager`, the `<instance_name>` would be `gitops-manager` and not `argocd`.